lib/jsdom/living/custom-elements/CustomElementRegistry-impl.js

```diff
@@ -25,7 +26,7 @@ function convertToSequenceDOMString(obj) {
     throw new TypeError("Invalid Sequence");
   }
 
-  return Array.from(obj).map(webIDLConversions.DOMString);
+  return Array.from(obj, v => webIDLConversions.DOMString(v));
```

Comment:
Please don't make unrelated changes in the PR. You can file issues asking if they are OK but it is not good to do so unsolicited.